### ISSUE TITLE
got versioning issues

### ISSUE THREAD

**Issue by jscook** [genehack/app-gitgot#48]
- $ got help

does not list 'version' as an available command

+ $ got version

reports what I expect

- $ got --version

does not report what I expect:

jsc@luskan Documents % got --version
/Users/jsc/.plenv/versions/5.20.1/bin/got
(Getopt::Long::GetOptions version 2.42; Perl version 5.20.1)

**Comment by genehack** [comment]
So, I can confirm 'version' does not appear in 'got help'. I'll try to figure out why that is. 

However, I can not confirm the differing version numbers, and I suspect you may have dueling Got versions installed, maybe? 

    % got version
    1.23
    
    % got --version
    got (App::GitGot) version 1.23 (/opt/plenv/versions/5.20.2/bin/got)

**Comment by jscook** [comment]
Sorry. That was from the previous version I had installed (1.22). The TRIAL version reports both versions correctly.

**Comment by genehack** [comment]
So, I've removed my local `version` subcommand lib. That means the `version` subcommand shipped with `App::Cmd` will handle both `got version` and `got --version`, so you'll see the same thing for both (once v1.24 is released).

I've also opened an issue over in `App::Cmd` land to see if there's any way to get the `version` command included in the command list -- right now, it's just stripped out, and there's no good way to override that behavior. That will be discussed here -> https://github.com/rjbs/App-Cmd/issues/45
